src/Hangfire.Mongo/MongoStorageOptions.cs

```diff
@@ -38,6 +38,16 @@ public MongoStorageOptions()
         /// </summary>
         public string Prefix { get; set; }
 
+        /// <summary>
+        /// Identify if using Hangfire with Cosmos DB with Mongo API
+        /// </summary>
+        public bool UseForCosmosMongoApi { get; set; }
+
+        /// <summary>
+        /// Comos TTL for "signals" and "notifications" collections
+        /// </summary>
+        public int CosmosHourlyTtl { get; set; }
+
```

Comment:
please inherit from MongoStorageOptions instead, so we dont change this file.
eg. CosmosMongoStorageOptions : MongoStorageOptions

Reply:
great suggestion, I created it.